lib/tasks/data/task.dart

```diff
@@ -3,7 +3,8 @@ import 'dart:ui';
 import 'package:flutter/material.dart';
 
 class Task {
-  final String id;
+  static int counter = 0;
+  final int id = counter++;
```

Comment:
The `id` should not simply be a running counter. The fact that it is a static property of the `Task` class is most likely going to be an issue when we look into saving tasks. Consider calculating the `id` e.g. based on hash values of the task's title, subtasks, due date etc.